app/views/directives/_build-pipeline-expanded.html

```diff
@@ -30,7 +30,7 @@
     </div>
     <div class="pipeline">
       <div class="pipeline-stage" ng-repeat="stage in jenkinsStatus.stages track by stage.id">
-        <div column class="pipeline-stage-column">
+        <div class="pipeline-stage-column">
```

Comment:
Shouldn't there be a `.pipeline-stage-column` rule with `flex-direction:column; display:flex` since it was removed from the markup?

Reply:
To be completely parallel with what was there, yes.  But a div displays block by default, and that renders the same way as flex/column, so I opted to remove it for less code.